Fix #4185 - Extend staff username maxlength to 64

plenaerts opened the pull request:
This change increases the ost_staff table username field length to 64.
The user forms do not warn against too long usernames.

Commit:
Fix #4185 - Extend staff username maxlength to 64

This change increases the ost_staff table username field length to 64.
The user forms do not warn against too long usernames.

JediKev commented:
@plenaerts This will fix the length for _new_ installations but this will not fix the length for _current_ installations or _upgrades_. You might want to look into adding a fix for current installs and upgrades. Cheers.

Commit:
Fix #4185 - Extend staff username maxlength to 64 for upgrades

This change increases the ost_staff table username field length to 64
for deploy upgrades.

Commit:
set schema sig manually in config

Commit:
Fix version, title, signature tags in upgrade sql

plenaerts commented:
@JediKev I think these commits cover upgrades.
